tests/dummy/app/router.js

```diff
@@ -62,6 +62,9 @@ Router.map(function() {
     this.route('prereg');
     this.route('allnodes');
     this.route('usernodes');
+    this.route('guid', {
+        path: '/:guid'
```

Comment:
A small suggestion: this effectively functions as a catch-all route (picking up things like `/profile`), so as part of this we may want an error event that automatically tries the OSF with the same URL if a matching GUID segment is found.

That will be especially useful during incremental migration when some but not all resource types have been implemented in ember- if we link to a resource type it can''t handle, we'll let the OSF decide which page to serve.